gui/src/buffer_previewer.cpp

```diff
@@ -52,7 +52,7 @@ BufferPreviewer::BufferPreviewer(int pixelsPerPeriod, double wavePhase, QWidget
 	, m_cursorVisible(true)
 {}
 
-BufferPreviewer::~BufferPreviewer() { delete[] m_wavePoints; }
+BufferPreviewer::~BufferPreviewer() { delete m_wavePoints; }
```

Comment:
This alone doesn't fix the crash. m_wavePoints needs to be initialized in constructor
m_wavePoints(new QPointF)